etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiPartnerMetadataConverter.java

```diff
@@ -22,10 +22,16 @@ public FhirMetadata<?> extractPublicMetadataToOperationOutcome(
         var operation = new OperationOutcome();
 
         operation.setId(requestedId);
-        operation.getIssue().add(createInformationIssueComponent("sender name", metadata.sender()));
         operation
                 .getIssue()
-                .add(createInformationIssueComponent("receiver name", metadata.receiver()));
+                .add(
+                        createInformationIssueComponent(
+                                "sender name", metadata.sendingFacilityDetails().namespace()));
```

Comment:
Same here. We should probably either have all properties in `MessageHdDataType`, or use `universalId`. The name should also probably change to either `sender` or `sender id`

Reply:
yes